data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -10,8 +14,41 @@
     settings.APP_ROOT / "tests" / "sources" / "national_risk_index" / "data"
 )
 
+"""
+Set UPDATE_TEST_FIXTURES to `True` temporarily to allow you to quickly update the test
+fixtures based on intentional changes to the logic of the method.
+
+However, note a few things first:
+
+1. Do *not* update these fixtures if you did not expect the ETL results to change.
+
+2. This should never be set to `True` in committed code. It should only be used
+temporarily to quickly update the fixtures.
+
+3. If the source data itself changes (e.g., the external source renames a column),
+update the "furthest upstream" test fixture which, in this case, is the .zip file.
+Then running `UPDATE_TEST_FIXTURES = True` will update all subsequent files.
+
+If you're confused by any of this, ask for help, it's confusing :).
+"""
+UPDATE_TEST_FIXTURES = False
+
 
 class TestNationalRiskIndexETL:
+    def test_update_test_fixtures(self, mock_paths, mock_etl):
+        """Assert that UPDATE_TEST_FIXTURES is False."""
```

Comment:
Overall these are really great flags, thanks. And I appreciate your care for detail and the extremely appropriate paranoia about what a future distracted developer might do with this! 

A few notes. 

Overall my philosophy here was to try to replicate the philosophy of [snapshot testing](https://blog.bitsrc.io/whats-snapshot-testing-and-can-we-use-it-for-frontend-f5a441cae27a) from the front-end world for backend ETL scripts. (Now that I'm talking about it, I realize that I 100% stole this idea from our friend @dfkoh !) Front-end snapshots will update with a single button press, and thus indeed it's true that the code under test will create the updated test output. 

With front-end snapshot testing, the approach tries to balance two needs: 

1. Alert the developer that their code has caused something to change in the site rendering, which may be intentional or unintentional. 
2. Make the tests easy enough to update that developers actually maintain good code coverage. 

In addition, by making the changes in the snapshot part of the codebase, any changes in outputs are subject to peer review. 

While updating the NRI tests for various small changes, I found myself getting fairly tired of how much work it takes to update the fixtures. (We have a similar challenge with https://github.com/usds/justice40-tool/issues/1048.) 

I'd love to have a way that easily updates the fixtures *after* I've carefully reviewed them to make sure that the outputs are indeed as expected. This also shows up in the `git diff` for a pull request. But as with front-end testing, to quote that link above, "At first many developers tend to blindly update the snapshots when they come across a failing test. However, blindly updating snapshots should never be done." 

I like the idea of making this a common method! And also maybe it shouldn't be a `test_` method at all, but instead a non-test method that gets run explicitly by the developer. Hmm.

Reply:
@lucasmbrown-usds - strongly agree on this:

> "At first many developers tend to blindly update the snapshots when they come across a failing test. However, blindly updating snapshots should never be done."

Given how simple it is to update snapshots on the FE, under deadlines, I am guilty of this (on other projects). However reading failures make take a few extra minutes and worth the extra time to understand why it's failing rather than blindly updating all snapshots.

I also would say that updating ALL snapshots versus updating the one that is failing is also a good practice.

Reply:
This is such a cool strategy! Doesn't surprise me that snapshot tests for backend code is David's brainchild haha. I'll definitely have to adopt them as a part of my testing toolkit.